server/integration-test/src/test/java/com/oceanbase/odc/service/notification/NotificationServiceTest.java

```diff
@@ -168,7 +168,11 @@ public void test_BatchUpdatePolicies() {
 
         List<NotificationPolicy> policies = notificationService.listPolicies(PROJECT_ID);
         Assert.assertFalse(policies.get(0).isEnabled());
-        Assert.assertEquals(channel.getId(), policies.get(1).getChannels().get(0).getId());
+        for (NotificationPolicy policy : policies) {
```

Comment:
invalid validation, what  if there is no policy contains metadataId==2?

Reply:
done, modified